back-end/app/controllers/artists_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ArtistsController < ApplicationController
+  before_filter :authenticate_user_from_token
+  def index
+    if current_user
+      render nothing: true
+    else
+      render json:{}, status: :unauthorized
+    end
+  end
```

Comment:
just for consistency it would be good to have a space before end like the above controller